guide/spanish/ruby/ruby-on-rails/index.md

```diff
@@ -4,9 +4,9 @@ localeTitle: Ruby on Rails
 ---
 # Ruby on Rails
 
-[Ruby on Rails](http://rubyonrails.org/) es un marco del lado del servidor (gema) construido en el lenguaje Ruby para hacer sitios web. Rails hace que el desarrollo web sea más rápido, más fácil y más divertido. Incluye todo lo que necesitas para crear aplicaciones fantásticas y tiene una gran comunidad. Rails fue creado por David Heinemeir Hansson y actualmente se encuentra en su quinta versión. Rails enfatiza el uso de otros patrones y paradigmas de ingeniería de software conocidos, que incluyen la [convención sobre la configuración (CoC)](https://en.wikipedia.org/wiki/Convention_over_configuration) , [no se repita (DRY)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) y el patrón de registro activo. Rails es un marco [modelo-vista-controlador (MVC)](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) , que proporciona estructuras predeterminadas para una base de datos, un servicio web y páginas web. Últimamente, Rails ha integrado un módulo API para que la creación de servicios web sea más rápida y fácil.
+[Ruby on Rails](http://rubyonrails.org/) es un framework del lado del servidor (gema) construido en el lenguaje Ruby para hacer sitios web. Rails hace que el desarrollo web sea más rápido, más fácil y más divertido. Incluye todo lo que necesitas para crear aplicaciones fantásticas y tiene una gran comunidad. Rails fue creado por David Heinemeir Hansson y actualmente se encuentra en la version 5.2.1 (https://weblog.rubyonrails.org/2018/8/7/Rails-5-2-1-has-been-released/). Rails enfatiza el uso de otros patrones y paradigmas de ingeniería de software conocidos, que incluyen la [convención sobre la configuración (CoC)](https://es.wikipedia.org/wiki/Convenci%C3%B3n_sobre_configuraci%C3%B3n) , [no te repitas (DRY)](https://es.wikipedia.org/wiki/No_te_repitas) y el patrón de registro activo. Rails es un marco [modelo-vista-controlador (MVC)](https://es.wikipedia.org/wiki/Modelo%E2%80%93vista%E2%80%93controlador) , que proporciona estructuras predeterminadas para una base de datos, un servicio web y páginas web. Últimamente, Rails ha integrado un módulo API para que la creación de servicios web sea más rápida y fácil.
```

Comment:
Specifying an exact version is bad, as it will be obsolete soon and we don't want to keep updating the article. Please revert his part of the change.